src/repositories/loginRegister.repository.ts

```diff
@@ -0,0 +1,27 @@
+import { QueryResult } from "pg";
+import { pool } from "../database/connection";
+import LoginRegisterRepoInterface from "./LoginRegisterRepoInterface";
+
+export default class LoginRegisterRepository implements LoginRegisterRepoInterface{
+
+    async checkEmail (email: string): Promise<QueryResult> {
+        return await pool.query("SELECT * FROM users WHERE email=$1", [email]);
```

Comment:
I am starting to be an asshole just to warn you, SELECT * is second worst query in the world, first one is query with LIKE operator in WHERE CLAUSE. Always write columns that you need, even if you need all of them.

Reply:
You are right there, it's not good practice